custom_components/presence_simulation/__init__.py

```diff
@@ -290,16 +274,10 @@ async def simulate_single_entity(entity_id, hist, overridden_delta, overridden_r
 
             #a while with sleeps of _interval_ seconds is used here instead of a big sleep to check regulary the is_running() parameter
             #and therefore stop the task as soon as the service has been stopped
-            target_time = state.last_updated + timedelta(overridden_delta)
-            if is_first:
```

Comment:
You can still use this functionality of not using the random delta for the first element of the `hist` collection if you want

Reply:
It looks like because of RecorderRun limit, that you're not guaranteed that the first hist entry will be the initial state.  Empirically, it looks like the first entry has its last_updated overridden to be the start_time, but I'm not sure we can rely on that.  Shall I not use the random_delta for the first entry anyways?
Either way is fine with me.

Reply:
Okay, understood. 
I guess it should be possible to know if the first element of the list is the initial state, somehow.
In the meanwhile, I think it is a good thing to not use the random for the first element. My reflection:
- Generally, for 'normal' users (not dev), HA will not be restarted frequently so it should work as expected
- If the first element is not the original state, it just means that we will not have a random offset for the first event, really not an issue from my point of view.

What do you think?

Reply:
K. I updated the diff to skip jitter on the first state update.